configure.ac

```diff
@@ -251,6 +251,8 @@ AC_CHECK_HEADERS(CoreServices/CoreServices.h, [
 ])
 AC_CHECK_FUNCS(backtrace backtrace_symbols backtrace_symbols_fd)
 
+AC_CHECK_FUNCS_ONCE(sys_siglist)
```

Comment:
But it seems I could remove `_ONCE` and have no effect.
https://www.gnu.org/software/autoconf/manual/autoconf-2.67/html_node/Generic-Functions.html